Sofa/Component/Collision/Response/Contact/src/sofa/component/collision/response/contact/DefaultContactManager.h

```diff
@@ -59,30 +59,6 @@ public :
         if (arg)
             obj->parse(arg);
 
-
-        //SOFA_ATTRIBUTE_DISABLED("v21.12 (PR#2522)", "v22.06","This attribute was only added to build a compatibility layer on the response name.")
-        {
-            static const std::map<std::string,std::string> renamingResponseMethod = {
```

Comment:
I would keep this one. To me 1 years compatibility layer for "scenes" is too short. It is very common to try to revive old project.

Reply:
I agree